sessionctx/variable/sysvar.go

```diff
@@ -677,6 +677,16 @@ var defaultSysVars = []*SysVar{
 			return nil
 		},
 	},
+	{Scope: ScopeGlobal, Name: TiDBStatsCacheMemQuota, Value: strconv.Itoa(DefTiDBStatsCacheMemQuota),
+		MinValue: 0, MaxValue: 1024 * 1024 * 1024 * 1024, Type: TypeInt,
```

Comment:
Why this MaxValue? If you want to define a meaningful maxValue, what about making it a const?

Reply:
updated.